docs/install/knative-with-any-k8s.md

```diff
@@ -411,6 +412,54 @@ The following command enables the default Broker on a namespace (here `default`)
 
 {{< /tab >}}
 
+{{% tab name="Github Source" %}}
+The following command installs the Github Source:
+
+   ```bash
+   kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/github.yaml
+   ```
+
+To learn more about the Github source, try [our sample](../eventing/samples/github-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Camel Source" %}}
+The following command installs the Camel Source:
+
+   ```bash
+   kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/camel.yaml
+   ```
+
+To learn more about the Camel source, try [our sample](../eventing/samples/apache-camel-source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="Kafka Source" %}}
+The following command installs the Kafka Source:
+
+   ```bash
+   kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/kafka-source.yaml
+   ```
+
+To learn more about the Kafka source, try [our sample](../eventing/samples/kafka/source/README.md)
+
+{{< /tab >}}
+
+{{% tab name="AWS SQS Source" %}}
+The following command installs the AWS SQS Source:
+
+   ```bash
+   kubectl apply --filename https://github.com/knative/eventing-contrib/releases/download/{{< version >}}/awssqs.yaml
+   ```
+
+<!-- TODO: Link to docs on how to use AWS SQS Source -->
```

Comment:
@n3wscott you touched this source last when it moved.  There's no sample for it, and no OWNERS in contrib.

I'm wondering if we should drop this until there's sample coverage and a clear owner?

Reply:
Ok, so discussed in #eventing-sources the criteria for inclusion here and the need for a sane way to check on the health of sources.  I'm dropping this because it's below that bar, and it can come back when it checks all the boxes (needs a sample).